### ISSUE TITLE
Some SVG lack a fill/stroke style and are invisible on the viewer (Firefox 54)

### ISSUE THREAD

**Issue by yhoyhoj** [FirefoxUX/icons#27]
Screenshot (on Firefox 54.0.1):
![screenshot](https://user-images.githubusercontent.com/2287301/27885314-d8c561b0-61d7-11e7-9714-4733f5c91a54.png)
  
It looks like the "header" lacks a stroke or fill attribute making these SVG transparent on the viewer.
Example (http://design.firefox.com/icons/icons/new/arrowhead-right-16.svg):
```xml
<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
```

**Comment by bwinton** [comment]
Oh yeah…  Hey @nt1m, what do you think about pre-processing the icons when we build the site?  Could that also make it easier to switch out the colours in the preview pane?

**Comment by nt1m** [comment]
@bwinton I'm not a fan of preprocressing SVGs tbh.

This is a bug in Firefox 54 that has been fixed in Firefox 55: https://bugzilla.mozilla.org/show_bug.cgi?id=1352258

You could try adding: 

```css
-moz-context-properties: fill;
fill: black;
```

to `.icon-display img` to workaround it.

**Comment by yhoyhoj** [comment]
The [-moz-context-properties](https://developer.mozilla.org/en-US/docs/Web/CSS/-moz-context-properties#Browser_compatibility) was added in Firefox 55 (and behind a flag) too so it won't help as a quick fix.  
  
As all the older icons have an explicit stroke/fill style (and the README show a "header" with a fill style), I figured it was an oversight.  
I just wanted to let you know but I don't really need to see those icons. If no one else complains, it can stay like this.
